pidfile/pidfile_test.go

```diff
@@ -0,0 +1,61 @@
+// +build !windows
```

Comment:
I guess this skip is due to missing implementation of `existsPid`, but even so it's better there's some tests for Windows (maybe in another file like `pidfile_windows_test.go`).